lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
What do you think, should we inline the package's contents instead of depending on it? It seems rather simple and I'm just generally scared of low-profile dependencies.